internal/website/content/_index.md

```diff
@@ -9,9 +9,6 @@ The Go Cloud Development Kit (Go CDK) is an open source project building
 libraries and tools to improve the experience of developing for the cloud with
 Go.
 
-{{< figure src="go-cdk-logo-gopherblue-small.png" alt="Go CDK logo"
```

Comment:
Where did the logo go?

Reply:
oh you're saying it went to the sidebar...

Reply:
Yup.